### DISCUSSION TITLE
Skywalking 9.7 stores fewer indexes into elasticsearch than mysql tables

### DISCUSSION THREAD

**Discussion by EmyiaEllen** [apache/skywalking#12060] [Q&A] [answered]
Hello, I'm learning to use skywalking and trying to do some customization work, like a simple tenant display.

In order to achieve, My plans are filter the query result by skywalking and directly send condition query to elasticsearch's skwalking indexes

I deploy es container on remote server and start skywalking oap-server and web-ui on my computer, when I search the indexes of skywalking, I found it's fewer than mysql generates.

![es](https://github.com/apache/skywalking/assets/82955455/64c3859a-4ef6-4935-9c7a-4638eba18691)

![image](https://github.com/apache/skywalking/assets/82955455/bd8f4a3b-93df-46b5-895c-7f25000af51b)

I only modified `storage.selector=${SW_STORAGE:elasticsearch}` in the application.yaml, and I want to directly search from indexes like `instance_traffic`, `metrics_cmp`, `metrics_count` and so on, Could you give me some help?

---
Skywalking-APM: 9.7.0
Skywalking-Agent: 9.7.0
Elasticsearch & kibana: OSS-7.4.2

**Comment by wu-sheng** [comment, marked as answer]
No, there's such indices. They are all in metric index.

**Reply by EmyiaEllen** [reply]
Thanks for your reply, I compare the skywalking version between two servers and immadiately noticed 9.1 is differenet from now 9.7. I re-read the changelog file and found:

> 9.2
> [Breaking Change] Elasticsearch storage merge all metrics/meter and records(without super datasets) indices into one physical index template metrics-all and records-all on the default setting.

And then I rebuild the elasticsearch container, and install skywaking-9.1, after start the oap-service, I query the indexes list and got I need

**Reply by wu-sheng** [reply]
You could use any version you like. But we abandon that for very certain reason. It has clear side effect about the performance.
And remember, once you picked an old release, you are on your own. The upstream will not be able to support you in any way.
